gateway/Roverfile.lock

```diff
@@ -1,4 +1,4 @@
-argparse 0.5.0-1||production
+argparse 0.6.0-1||production
```

Comment:
Do we need the new version for some reason?

Reply:
No good reason. rover is not so smart about releasing locked versions.